src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs

```diff
@@ -1,3 +1,5 @@
+using Nancy.Configuration;

```

Comment:
Why do we have to use all other conventions than what's out of the box? Geez. And no .DotSettings :cry: